cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -297,19 +299,26 @@ public void getAllApps(HttpRequest request, HttpResponder responder,
 
   private ScanApplicationsRequest getScanRequest(String namespaceId, String artifactVersion, String pageToken,
                                                  Integer pageSize, SortOrder orderBy, String nameFilter,
-                                                 Set<String> names) {
+                                                 Set<String> names, String latestOnly, String strictEqual) {
```

Comment:
should we have `strictEqual` that is dependent on `nameFilter`? Or an exclusive parameter instead?

Reply:
the current implementation is even if you set strictequal without namefilter, it will not do anything

Reply:
yeah but `strictEqual` works only when we have `nameFilter` right? Do we want that?

Reply:
yea, if there's no nameFilter then it's returning everything, so strictEqual is irrelevant